client/src/components/ui/LandingPage.tsx

```diff
@@ -1,12 +1,52 @@
-import Navbar from "./Navbar";
-import { Container } from "@chakra-ui/react";
+import { ChakraProvider, extendTheme, VStack, Text, CSSReset } from "@chakra-ui/react";
+import "css/LandingPage.css";
 
-const LandingPage = () => {
+const theme = extendTheme({
+  fonts: {
+    heading: `'Michroma'`,
+    body: `'Michroma'`,
+  },
+  colors: {
+    primary: {
+      100: "#D3DEE1",
+      900: "#00293F",
+    },
+  },
+  styles: {
+    global: {
+      "html, body": {
+        height: "100%",
+        margin: 0,
+        padding: 3,
+      },
+      body: {
+        bgGradient: "linear(to-b, primary.900, 55%, primary.100)",
+      },
+    },
+  },
+});
+
+export default function App() {
   return (
-    <Container minH="100vh" style={{ backgroundColor: "blue" }}>
-      <Navbar />
-    </Container>
+    <ChakraProvider theme={theme}>
```

Comment:
In ChakraUI, themes are global. So the styles you put under `global` will apply for every page in the entire app, even when this component is unmounted.

I don't recommend putting anything under `global` in the theme styles unless you intend for it to affect all pages in the app.

Reply:
Giving this component's `VStack` a height of `100vh` is good enough and would avoid interfering with other parts of the app.